README.md

```diff
@@ -35,6 +36,7 @@ Config is stored in your `localStorage`.
 - `token`: GitHub token.  You can create one in your settings [page](https://github.com/settings/tokens).  During creation of token enable `repo` in scope section.  If you like to display PR from private repositories you will have to authenticate by clicking on `sso` button after token is created.
 - `refreshTime`: Time in milliseconds to next refresh.  On each main refrest list of PR is updated.  If new PR appear then it will be added to list.  From now PR is refreshed independent from main list.  Time to next refresh is displayed on top of the page (for PR list) and in left bottom corner of each PR.
 - `defaultBranch`: It's name of branch PR is aiming to mere.  If someone accidental create PR with different target then you will see red warning in top left corner with branch name.
+- `ticketsUrl`: Link to your issue tracker.  To make it wokr you have to begin your PR with ticket ID like: `PROJECTNAME-NUMBER` or `TEAMNAME NUMBER` (so its text and number separated with space or dash).
```

Comment:
To make it wokr? And double space earlier ?

Reply:
Type fixed.

-----

I use double spaced after period in monospace fonts (it's like that in whole file). Monospace font have same space between letters. In that case double space after period makes text more readable. **It's different for proportionals fonts.** With proportional fonts single space is enough because spacing and kerning is controlled by font.

Double space was commonly used by typewriters because they too used monospace font. Convention is still alive in computers. If you check `man` document's in UNIX you will find that most of them are written with double space after period as well. Try `man cat`, `man ls`, `man pwd` etc. in Your terminal.

[Read more](https://graphicdesign.stackexchange.com/questions/2160/typesetting-two-spaces-after-the-period)

In html preview of markdown file It's displayed as single space anyway.